src/audio/dai-zephyr.c

```diff
@@ -317,7 +315,6 @@ dai_dma_cb(struct dai_data *dd, struct comp_dev *dev, uint32_t bytes,
 		audio_stream_consume(&dma_buf->stream, bytes);
 	}
 
-	local_buf = buffer_acquire(dd->local_buffer);
 	/* assert dma_buffer_copy succeed */
```

Comment:
Ok, this looked suspicious at first, but seeems we indeed unncessarily release and acquire local_buf here.

Reply:
yes, here should have a bug before the change.